app/models/slate_config.py

```diff
@@ -46,7 +46,7 @@ class SlateConfigModel:
     SLATE_CONFIGS_BY_ID = {}
 
     def __init__(self, slate_id: str, display_name: str, description: str,
-                 curator_topic_label: Optional[CuratorTopic] = None, experiments=None):
+                 curator_topic_label: Optional[str] = None, experiments=None):
```

Comment:
This fixes the type of this argument as it is used in production today. We're passing a string in here, not an enumation value from `CuratorTopic`.